docs/src/pages/formats/module-story-format/index.md

```diff
@@ -0,0 +1,77 @@
+---
+id: 'module-story-format'
+title: 'Module Story Format'
+---
+
+Module story format is the recommended way to [write stories](../../basics/writing-stories/) since Storybook 5.2.
+
+It's called "module" format because the stories and component metadata are defined as ES6 modules. Every Module story file consists of a required default export and one or more named exports.
```

Comment:
```suggestion
It's called "module" format because the stories and component metadata are defined as ES6 modules. Every module story file consists of a required default export and one or more named exports.
```